src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar{
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  gap: $gap-between-day;
+  padding: $padding;
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: $day-width;
+    height: $day-width;
+    box-sizing: border-box;
+
+    font-size: 30px;
+    color: $text-color;
+
+    background-color: $day-background-color;
+    border: 1px solid $text-color;
+    cursor: pointer;
+
+    &:hover{
+      background-color:  $hover-color;
+      transform: translateY(-20px);
+      transition:  0.5s all;
```

Comment:
Apply transition to the selector itself instead of just :hover and specify to which properties should apply transition instead of `all`